app/models/work_history.rb

```diff
@@ -65,4 +65,17 @@ def current_or_most_recent_role?
     application_form.work_histories.empty? ||
       application_form.work_histories.ordered.first == self
   end
+
+  def summary_title
```

Comment:
I think we should try and avoid having view logic in the models. Could this go in a view object (or helper?). Happy to approve as is if you want to card up a refactor given we need to this in.

Reply:
Agreed, I've got a list of follow up improvements I'd like to make to these PRs after the user research is done and I'll add this to the list.